controllers/products.js

```diff
@@ -1,15 +1,17 @@
 const productsModel = require("../models/products");
 
+const showAllProductsFunc = function(req, res, next) {
+	productsModel.find({}, function(err, products) {
+		if (err) {
+			next(err);
+		} else {
+			res.json({
+				products: products
+			});
+		}
+	});
+};
+
 module.exports = {
-	showAllProducts: function(req, res, next) {
-		productsModel.find({}, function(err, products) {
-			if (err) {
-				next(err);
-			} else {
-				res.json({
-					products: products
-				});
-			}
-		});
-	}
+	showAllProducts: showAllProductsFunc
```

Comment:
exporting with object can be done without the need of key and value pairs like this 
```
{
showAllProductsFunc
}
```
and implicitly the key will be showAllProductsFunc and the it's value will be the function 
and while importing you can rename the value if you want to like this
```
const { showAllProductsFunc : myNewFuncName } = require('file path')
```

Reply:
great thanks